Formula/glog.rb

```diff
@@ -12,11 +13,29 @@ class Glog < Formula
     sha256 "ccb8f5022bffec4a768851feb5a8cc1bf52ee7d9b8b87b8f9ce6bc4cd28278f9" => :yosemite
   end
 
+  depends_on "cmake" => :build
   depends_on "gflags"
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make", "install"
+    mkdir "glog-build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+    lib.install_symlink "libglog.dylib" => "libglog.0.dylib"
```

Comment:
My bad, this needs to be `lib/"libglog.dylib" => "libglog.0.dylib"`.  Could you also add a comment with a link to the upstream issue, so we know when to remove this line?